src/rard/research/tests/models/test_original_text.py

```diff
@@ -203,3 +204,29 @@ def test_no_edition_in_display(self):
         }
         citing_work = CitingWork.objects.create(**data)
         self.assertFalse(str(citing_work).find(data["edition"]) >= 0)
+
+
+class TestReferences(TestCase):
+    def setUp(self):
+        self.citing_work = CitingWork.objects.create(title="title")
+        fragment = Fragment.objects.create(name="fragment_name")
+
+        self.original_text = OriginalText.objects.create(
+            owner=fragment,
+            citing_work=self.citing_work,
+        )
+
+        reference = "2.6-9"
+        self.reference = Reference.objects.create(
+            editor="superfluous",
+            reference_position=reference,
+            original_text=self.original_text,
+        )
+
```

Comment:
I think we need a test_reference_list_property test here. So, make sure that when there's only one reference you just get the reference position, and when there's two you get "editor1 1.2.3 | editor2 4.5.6"